both/methods.js

```diff
@@ -228,11 +228,11 @@ Meteor.methods({
         registrationId: 1,
       },
     }).fetch();
-    if(competitorCount < 0) {
+    if(participantCount < 0) {
```

Comment:
I wrote this. I was thinking of this as just "get rid of the undefined operation". While useless, it's not actually nonsensical to advance 0 people, it is well defined. That said, I'm happy for this to change if you want to make it happen =). I don't know what our UI enforces.